packages/rolldown/src/plugin/bindingify-plugin.ts

```diff
@@ -126,7 +126,7 @@ export function bindingifyPlugin(
     bindingifyCloseWatcher(args)
 
   const result: BindingPluginOptions = {
-    name: plugin.name ?? 'unknown',
+    name: plugin.name!,
```

Comment:
Could you add a comment saying that nameless plugin would be surely assign a name by rolldown?

Reply:
Done